DEVELOPER.md

```diff
@@ -0,0 +1,30 @@
+# Developer's Guide
+
+Welcome to the development reference for Camunda's C8 Multi Region! This document provides guidance on setting up a testing environment, running tests, and managing releases.
+
+## Setting up Development Environment
+
+TODO: this part is not yet documented outside of the official documentation: https://docs.camunda.io/docs/next/self-managed/setup/deploy/amazon/amazon-eks/dual-region/
+
```

Comment:
TBH, I don't have a good developer setup besides the guide.
Overall, one can run the whole setup on a single cluster (even kind) without the whole DNS chaining etc. but at the end of the day, Elasticsearch will always be problematic as it needs a proper backup solution if one wants failback/failover.

Like we could describe a setup based on kind and host file system, just that I'm not sure the use justifies it.
If AWS remains the way to go.

1. Create a Kubernetes cluster (any flavour)
2. Adjust values in `camunda-values.yml` in `aws/dual-region/kubernetes` by utilizing the scripts `aws/dual-region/scripts/`
2.1. `create_elasticsearch_secrets.sh` and `generate_zeebe_helm_values.sh`
3. Deploy `aws/dual-region/kubernetes/region0` to namespace `camunda-primary`
```
helm install camunda camunda/camunda-platform -f camunda-values.yml -f region0/camunda-values.yml --create-namespace --namespace camunda-primary
```
4. Deploy `aws/dual-region/kubernetes/region1` to namespace `camunda-secondary`
```
helm install camunda camunda/camunda-platform -f camunda-values.yml -f region1/camunda-values.yml --create-namespace --namespace camunda-primary
```